lib/Console/Command/RunCommand.php

```diff
@@ -131,6 +131,10 @@ public function execute(InputInterface $input, OutputInterface $output)
 
         $suite = $this->runnerHandler->runFromInput($input, $output, $config);
 
+        if ($input->getOption('uuid') || $input->getOption('file')) {
+            $suite->mergeBaselines($this->suiteCollectionHandler->suiteCollectionFromInput($input));
+        }
```

Comment:
We can refactor to include the baselines in the model _before_ running the benchmarks. This would allow the assertions to take the baselines into account.